nodeadm/cmd/nodeadm/init/init.go

```diff
@@ -74,6 +75,19 @@ func (c *initCmd) Run(log *zap.Logger, opts *cli.GlobalOptions) error {
 		return err
 	}
 
+	log.Info("Configuring system aspects...")
```

Comment:
since this is still a "configuration" step can we put it in the `configPhase` block? it breaks the contract that we wont touch the filesystem after cloud-init runs user data

Reply:
Sure thing, I had if statement blindness